README.md

```diff
@@ -282,8 +367,8 @@ Before submitting a pull request, please use [format-diff.py](format-diff.py) to
 > **NOTE**: It is required to use clang-format 12.0.0+.
 
 ```shell
-cd $WORKSPACE/tiflash
-python3 format-diff.py --diff_from `git merge-base ${TARGET_REMOTE_BRANCH} HEAD`
+# In the TiFlash repository root:
+python3 format-diff.py --diff_from `git merge-base origin/master HEAD`
```

Comment:
It's may better to add up the clang-tidy check.

Reply:
Nice catch!

Reply:
After some investigation, I think we can introduce the clang-tidy-diff.py to run clang-tidy easier in later PRs. The current `release-centos7-llvm/scripts/run-clang-tidy.py` needs to be run along with a script to modify the compile database so that only changed files are linted, which is pretty hack. What do you think?